server/modules/protocol/MySQL/mysql_common.cc

```diff
@@ -1809,7 +1812,11 @@ void mxs_mysql_parse_ok_packet(GWBUF *buff, size_t packet_offset, size_t packet_
                 case SESSION_TRACK_STATE_CHANGE:
                 case SESSION_TRACK_SCHEMA:
                 case SESSION_TRACK_GTIDS:
-                    mxs_lestr_consume(&ptr, &size);
+                    mxs_leint_consume(&ptr);
```

Comment:
Adding comments that explain what parts of the packet these commands consume would be helpful.